diagnostics-app/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
same comment: let's remove editor-specific files from here. Maybe add them in the upper level .gitignore